### PR TITLE
DENG-2784 add ga4 sensors

### PR DESCRIPTION
**Pull request by kwindau**
This adds 2 wait tasks - one for the derived tables coming from the blogs events Google table, and one for the derived tables coming from the main mozilla website events Google table.

Checklist for reviewer:

- [x] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title).
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated.
- [ ] When adding a new derived dataset, ensure that data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data can be available in the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](https://github.com/mozilla/bigquery-etl/blob/main/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-2817)

### CODE REVIEW

**Comment by lelilia** [comment]
Wondering if this is really [`depends_on_tables_existing`](https://github.com/mozilla/bigquery-etl/blob/7c779c4f9024b9ae9093c24b61371f921a413ca9/tests/query_scheduling/test_dag_collection.py#L780) or rather [`depends_on_table_partitions_existing`](https://github.com/mozilla/bigquery-etl/blob/7c779c4f9024b9ae9093c24b61371f921a413ca9/tests/query_scheduling/test_dag_collection.py#L790)?

**Comment by kwindau** [comment]
> Wondering if this is really [`depends_on_tables_existing`](https://github.com/mozilla/bigquery-etl/blob/7c779c4f9024b9ae9093c24b61371f921a413ca9/tests/query_scheduling/test_dag_collection.py#L780) or rather [`depends_on_table_partitions_existing`](https://github.com/mozilla/bigquery-etl/blob/7c779c4f9024b9ae9093c24b61371f921a413ca9/tests/query_scheduling/test_dag_collection.py#L790)?

@sean-rose - can you let me know which one you recommend for a sharded table?  I wasn't sure, went with "depends_on_tables_existing" - but let me know if I should switch it

File: sql/moz-fx-data-marketing-prod/ga_derived/www_site_landing_page_metrics_v2/metadata.yaml
Comment (sean-rose) [inline sql/moz-fx-data-marketing-prod/ga_derived/www_site_landing_page_metrics_v2/metadata.yaml:13]: This ETL doesn't directly use the GA4 tables, so it doesn't need this dependency defined.
```suggestion
```

File: sql/moz-fx-data-marketing-prod/ga_derived/www_site_page_metrics_v2/metadata.yaml
Comment (sean-rose) [inline sql/moz-fx-data-marketing-prod/ga_derived/www_site_page_metrics_v2/metadata.yaml:13]: This ETL doesn't directly use the GA4 tables, so it doesn't need this dependency defined.
```suggestion
```

File: sql/moz-fx-data-shared-prod/mozilla_org_derived/ga_clients_v2/metadata.yaml
Comment (sean-rose) [inline sql/moz-fx-data-shared-prod/mozilla_org_derived/ga_clients_v2/metadata.yaml:16]: This ETL doesn't directly use the GA4 tables, so it doesn't need this dependency defined.
```suggestion
```

File: sql/moz-fx-data-marketing-prod/ga_derived/blogs_daily_summary_v2/metadata.yaml
Comment (kwindau) [reply sql/moz-fx-data-marketing-prod/ga_derived/blogs_daily_summary_v2/metadata.yaml:13]: That's a good point, thank you!

File: sql/moz-fx-data-marketing-prod/ga_derived/www_site_downloads_v2/metadata.yaml
Comment (kwindau) [reply sql/moz-fx-data-marketing-prod/ga_derived/www_site_downloads_v2/metadata.yaml]: I personally like having it this way because it's easy to read exactly how it's calculated directly from the source without having to go to an intermediate which kind of obfuscates what is happening, so I hope it's ok if I leave it like this for now!

File: sql/moz-fx-data-marketing-prod/ga_derived/www_site_hits_v2/metadata.yaml
Comment (kwindau) [reply sql/moz-fx-data-marketing-prod/ga_derived/www_site_hits_v2/metadata.yaml]: done, thanks!

File: sql/moz-fx-data-marketing-prod/ga_derived/www_site_downloads_v2/metadata.yaml
Comment (sean-rose) [reply sql/moz-fx-data-marketing-prod/ga_derived/www_site_downloads_v2/metadata.yaml]: That's fine, since there doesn't appear to be much duplicated logic in this case.

**Review by sean-rose — APPROVED** [review]
This will need to wait until #5100 is merged (and then merge `main` after that), because as you can see in the SQL diff the generated `bqetl_google_analytics_derived_ga4` DAG would currently fail due to the `wait_for_blogs_events_table` and `wait_for_wmo_events_table` variables being referenced before they exist.

**Pull request merged**
